src/jit/morph.cpp

```diff
@@ -1439,6 +1439,14 @@ void fgArgInfo::ArgsComplete()
             continue;
 #endif
         }
+#if defined(_TARGET_ARM_) && !defined(LEGACY_BACKEND)
+        // Not tested on LegacyJIT/ARM32 yet
+        else if (curArgTabEntry->isSplit)
```

Comment:
This will never be used on arm32 `LEGACY_BACKEND` -- we shouldn't be touching that code.